src/components/cards/ReceiveAssetsCard.tsx

```diff
@@ -42,12 +42,12 @@ export const ReceiveAssetsCard = () => {
     [accountAddress, isToastActive, setClipboard, setToastActive]
   );
 
-  const onPressQRCode = useCallback(() => {
+  const onPressQRCode = () => {
```

Comment:
Just curious, why remove these useCallbacks?

Reply:
i don't see why having them is beneficial